source/scss/1200.scss

```diff
@@ -1,15 +1,39 @@
-.menu {
-    &.main {
-        li {
-            a {
-                font-size: 2em;
+header {
+
+  .nav-main {
+   
+    nav {
```

Comment:
Can you reduce the specificity here? There's only one nav in the header so it can just be
```
header {
	nav {
		li {}
		a {}
	}
}
```

Reply:
done